api-reference/_hidden/GridBaseColumn/filterOperations.md

```diff
@@ -30,6 +30,22 @@ The following table lists available filters by [data types](/api-reference/_hidd
   </tr>
 </table>
 
+The following table contains filter operations specific for the nested filter builder:
```

Comment:
As I understand, some filter operations can be used in both the filter row and the filter builder and other filter operations can be used in the filter builder only. If that is so, then this thought should be articulated more clearly.

Reply:
```suggestion
The following table contains filters that users can apply only in the nested filter builder:
```

Reply:
```suggestion
The following table contains filter operations that users can apply only in the nested filter builder:
```